admin-dev/themes/default/css/bundle/module/module.css

```diff
@@ -249,27 +250,38 @@ img.module-logo-thumb-list {
     vertical-align: top;
 }
 .module-modal-title > h4 {
-    color: #1E2024 !important;
-    font-family: "Open Sans", Helvetica, Arial,sans-serif !important;
-    font-weight: bold !important;
-    font-size: 14px;
+    font-family: "Open Sans Semibold", Helvetica, Arial,sans-serif !important;
```

Comment:
Hi @kpodemski,

We have this font loaded so I went straight forward using this instead of implementing what you explained in your post. Just that :)

Have a good day Sir :)